projects/jsoncpp/Dockerfile

```diff
@@ -0,0 +1,33 @@
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+MAINTAINER cdunn2001@gmail.com
+RUN apt-get update && apt-get install -y make python
+RUN apt-get update && apt-get install -y make wget tshark
```

Comment:
Why is `make` installed twice?
Can't `wget` and `tshark` be installed on the previous line?
Why does this need tshark?

Reply:
Yea, no need for tshark. Will rewrite this section.